WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/refunds/IssueRefundViewModel.kt

```diff
@@ -268,6 +268,13 @@ class IssueRefundViewModel @Inject constructor(
             .filter { refundableFeeLineIds.contains(it.feeLine.id) }
         _refundFeeLines.value = feeLines
 
+        if (feeLines.isNotEmpty() && items.isEmpty()) {
```

Comment:
nitpick: ❓ I believe it would be beneficial to encapsulate the condition `feeLines.isNotEmpty() && items.isEmpty()` within a distinct method, such as `orderContainsOnlyCustomAmounts`. This would enhance the readability and clarity of the code, providing a straightforward understanding of its purpose. What do you think?
```suggestion
        if (orderContainsOnlyCustomAmounts()) {
```

```
private fun orderContainsOnlyCustomAmounts(): Boolean = feeLines.isNotEmpty() && items.isEmpty()
```

Reply:
refactored here https://github.com/woocommerce/woocommerce-android/pull/10236/commits/7fc95339c9f44aadea929c6fd2982d75ec663161